### ISSUE TITLE
DELETE method is not supported

### ISSUE THREAD

**Issue by askucher** [masotime/mock-express#3]
```
app.delete('/test', function(req, res, next) {
    var model = { name: 'world'};

    if (req.query.start === 'true') {
        res.render('index', model);
    } else {
        res.redirect('http://www.google.com');
    }

});
```

**Comment by askucher** [comment]
Created a pull request

**Comment by masotime** [comment]
Merged https://github.com/masotime/mock-express/pull/4 and published as 0.1.0.
